fml/logging.cc

```diff
@@ -72,61 +72,75 @@ LogMessage::LogMessage(LogSeverity severity,
   }
 }
 
+// static
+std::shared_ptr<std::ostringstream> LogMessage::test_stream_ = nullptr;
```

Comment:
```suggestion
std::shared_ptr<std::ostringstream> LogMessage::capture_next_log_stream_ = nullptr;
```

It's probably better that the method name and the variable have the same verbiage.